lib/MusicBrainz/Server/Controller/Admin/WikiDoc.pm

```diff
@@ -33,14 +35,22 @@ sub index : Path Args(0)
                 $c->log->error("'$pages[$i]->{id}' from the transclusion table doesn't match '$wiki_pages[$i]->{id}' from the wiki");
             } else {
                 # Problem accessing the api data
-                $c->stash->{wiki_unreachable} = 1;
+                $wiki_unreachable = 1;
             }
         }
     }
 
+    my %props = (
+        pages             => \@pages,
+        updatesRequired   => boolean_to_json($updates_required),
+        wikiIsUnreachable => boolean_to_json($wiki_unreachable),
+        wikiServer        => $c->stash->{wiki_server},
+    );
+
     $c->stash(
-        pages            => \@pages,
-        updates_required => $updates_required
+      component_path => 'admin/wikidoc/WikiDocIndex',
+      component_props => \%props,
+      current_view => 'Node',
```

Comment:
Small nitpick: some of the `$c->stash` calls here and below are indented 2 spaces instead of 4.

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/039ca4facc953bc873427edb20f18e586593f3a7, https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/dd37036d4f606dc2df9814c690b9b3eeb28f802a, https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/983c361dc0b4c1a7e6ddccfdae077f237d95067a, https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/87cbb331d7ff4b26b90442dcca83e9256bce64c5.